modules/post-installation-adding-nutanix-failure-domains-compute-machines-replace.adoc

```diff
@@ -0,0 +1,223 @@
+====
++
+.Example compute machine set with Nutanix failure domains
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: MachineSet
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: <cluster_name>
+  name: <new_machine_set_name_1>
+  namespace: openshift-machine-api
+spec:
+  replicas: 2
+# ...
+  template:
+    spec:
+# ...
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          failureDomain:
+            name: <failure_domain_name_1>
+          cluster:
+            type: uuid
+            uuid: <prism_element_uuid_1>
+          subnets:
+          - type: uuid
+            uuid: <prism_element_network_uuid_1>
+# ...
+----
+
+. Save your changes.
+
+. Create a compute machine set CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <new_machine_set_name_1>.yaml
+----
+
+. As required, continue to create compute machine sets to reference the additional failure domains that are available to the deployment.
+
+. List the machines that are managed by the new compute machine sets by running the following command for each new compute machine set:
++
+[source,terminal]
+----
+$ oc get -n openshift-machine-api machines -l machine.openshift.io/cluster-api-machineset=<new_machine_set_name_1>
+----
++
+.Example output
+[source,text]
+----
+NAME                             PHASE          TYPE         REGION      ZONE         AGE
+<machine_from_new_1>             Provisioned    m6i.xlarge   us-west-1   us-west-1a   25s
+<machine_from_new_2>             Provisioning   m6i.xlarge   us-west-1   us-west-1a   25s
+----
++
+When the new machines are in the `Running` phase, you can delete the old compute machine sets that do not include the failure domain configuration.
+
+. When you have verified that the new machines are in the `Running` phase, delete the old compute machine sets by running the following command for each:
++
+[source,terminal]
+----
+$ oc delete machineset <original_machine_set_name_1> -n openshift-machine-api
+----
+
+.Verification
+
+* To verify that the compute machine sets without the updated configuration are deleted, list the compute machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
+[source,text]
+----
+NAME                       DESIRED   CURRENT   READY   AVAILABLE   AGE
+<new_machine_set_name_1>   1         1         1       1           4m12s
+<new_machine_set_name_2>   1         1         1       1           4m12s
+----
+
+* To verify that the compute machines without the updated configuration are deleted, list the machines that are managed by the updated compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc get -n openshift-machine-api machines -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
```

Comment:
maybe should not specify `-l machine.openshift.io/cluster-api-machineset=<machine_set_name>`, because it can only list the machines of this machine set, but in the example later it lists machines for two machine set.